needle-zendesk-connector/src/server/backend/connectors-backend.ts

```diff
@@ -0,0 +1,257 @@
+import type {
+  ConnectorRequest,
+  CreateConnectorRequest,
+} from "~/models/connectors-models";
+import {
+  createConnector,
+  getConnector,
+  listConnectors,
+  deleteConnector,
+  publishConnectorRun,
+  createNeedleFileId,
+  type Session,
+  type ConnectorRunDescriptor,
+} from "@needle-ai/needle-sdk";
+
+import { db } from "../db";
+import { filesTable, zendeskConnectorsTable } from "../db/schema";
+import { eq } from "drizzle-orm";
+import { createZendeskService } from "../zendesk/service";
+import {
+  getConnectorDetails,
+  getCurrentFiles,
+  handleDatabaseUpdates,
+} from "./zendesk-db";
+import { type DbFile } from "../zendesk/types";
+import { computeDiff } from "./diff-utils";
+
+export async function createZendeskConnector(
+  params: CreateConnectorRequest,
+  session: Session,
+) {
+  const connector = await createConnector(
+    {
+      name: params.name,
+      cronJob: "0 0 * * *",
```

Comment:
Let's make user select cronjob details as well. (We need to extend `CreateConnectorRequestSchema`)

Reply:
Can u share with me the code for the cronjob selector please?

Reply:
passed over Slack